src/main/resources/org/mal_lang/langspec/lang.schema.json

```diff
@@ -0,0 +1,554 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/mal-lang/mal-langspec/master/src/main/resources/org/mal_lang/langspec/lang.schema.json",
+  "title": "MAL Langspec",
+  "description": "",
+  "definitions": {
+    "identifier": {
+      "type": "string",
+      "pattern": "^[A-Za-z_][A-Za-z0-9_]*$"
```

Comment:
minLength needed, maybe MaxLength too? 256?

Reply:
minLength is not needed because of the pattern. I'm not sure we want to limit the length here.

Reply:
doesnt "^[A-Za-z_][A-Za-z0-9_]*$" match ""? shouldn't it in that case be "^[A-Za-z_][A-Za-z0-9_]+$" ?

Reply:
the `[A-Za-z_]` part matches exactly one character, and `[A-Za-z0-9_]*` matches 0+ characters, so 1+ characters in total